tests/const_dsm_5.py

```diff
@@ -3,10 +3,471 @@
 from .const import SERIAL, SID, UNIQUE_KEY  # pylint: disable=unused-import
 
 # DSM 5 RAW DATA
-DSM_5_LOGIN = {}
+DSM_5_LOGIN = {
+    {'data': {'sid': SID}, 'success': True}
+DSM_5_LOGIN = {'data': {'sid': SID}, 'success': True}
```

Comment:
Whooops for the bad suggestion lines

```suggestion
DSM_5_LOGIN = {'data': {'sid': SID}, 'success': True}
```